app/src/main/res/values/dimens.xml

```diff
@@ -97,4 +97,5 @@
     <dimen name="contest_result_entry_rank_textsize">14sp</dimen>
     <dimen name="contest_result_entry_image_width">88dp</dimen>
     <dimen name="contest_result_entry_image_height">100dp</dimen>
+    <dimen name="entry_checkmark_right_padding">7dp</dimen>
```

Comment:
we should bump this to 8. In general, Android recommends keeping dimensions in multiples of 8 and since its so close its worth bumping up. Also let's rename it end padding so it makes more sense in RTL languages.